edb/schema/scalars.py

```diff
@@ -370,6 +370,11 @@ def _cmd_tree_from_ast(
             create_cmd = cmd
 
         if isinstance(astnode, qlast.CreateScalarType):
+            if not astnode.bases:
+                raise errors.SchemaError(
+                    f'scalar type must have a supertype',
+                    context=astnode.context,
```

Comment:
Let's add a hint with an example, e.g. `hint="For example: scalar type {name} extending str"`